.env.example

```diff
@@ -28,6 +28,9 @@ ETL_DB_NAME=
 ETL_DB_PASSWORD=
 ETL_DB_PORT=5432
 ETL_DB_USER=
+GOOGLE_CREDENTIALS_JSON=
```

Comment:
@jessehon Just wanted to check - these env vars don't need to be in `app.json` because they're only loaded in the worker which doesn't matter on review apps, correct?

Reply:
Nope, just forgot to add them haha